Update use-effect.mdx (#2463)
yewstack/yew: website/docs/concepts/function-components/hooks/use-effect.mdx

`use_effect` is used for hooking into the component's lifecycle and creating side-effects.

> It takes a function which is called everty time after the component's render finished.

Reviewer: ```suggestion
It takes a function which is called everty time after the component's render is finished.
```

Author: Not so sure about that change. I would either use `has finished` or nothing. But it's up to you, it's really not that important

Reviewer: It's fine either way. Use "has" or "is", whichever you want. "render finished" sounds weird

> It takes a function which is called everty time after the component's render finished.

Author: ```suggestion
It takes a function which is called every time after the component's render finished.
```

> It takes a function which is called everty time after the component's render finished.

Reviewer 2: ```suggestion
It takes a function which is called every time after the component's render has finished.
```
